src/components/modal/delete-modal.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import {
+  k8sDeleteResource,
+  K8sModelCommon,
+  K8sResourceCommon,
+} from '@openshift/dynamic-plugin-sdk-utils';
+import {
+  Button,
+  ButtonType,
+  ButtonVariant,
+  Form,
+  ModalVariant,
+  Stack,
+  StackItem,
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+import { Formik } from 'formik';
+import { InputField } from '../../shared';
+import { ModalComponentProps, createModalLauncher } from './createModalLauncher';
+
+type DeleteResourceModalProps = ModalComponentProps & {
```

Comment:
Can you use the `ComponentProps` type from `createModalLauncher` here 

Would really appreciate it if you could do the same to the `BuildLogViewerProps` type, I missed it in my PR

Reply:
Updated.